lib/logger/logger.go

```diff
@@ -29,6 +32,16 @@ var (
 
 	errorsPerSecondLimit = flag.Int("loggerErrorsPerSecondLimit", 0, `Per-second limit on the number of ERROR messages. If more than the given number of errors are emitted per second, the remaining errors are suppressed. Zero values disable the rate limit`)
 	warnsPerSecondLimit  = flag.Int("loggerWarnsPerSecondLimit", 0, `Per-second limit on the number of WARN messages. If more than the given number of warns are emitted per second, then the remaining warns are suppressed. Zero values disable the rate limit`)
+
+	syslogNetwork = flag.String("syslogNetwork", "", "network connection to establish. Options are tcp udp tcp+tls. If network is empty, it will connect to the local syslog server")
```

Comment:
It would be great to make it consistent with TLS flags: `syslog.network`, `syslog.address`, `syslog.tag`